site/src/github/comparison_summary.rs

```diff
@@ -286,32 +281,6 @@ for rollup, we strongly recommend not doing so since this PR may lead to changes
 compiler perf.{next_steps}
 
 @bors rollup=never
-@rustbot label: +S-waiting-on-review -S-waiting-on-perf {label}",
+@rustbot label: +S-waiting-on-review -S-waiting-on-perf {sign}perf-regression",
```

Comment:
This is not relevant for this PR, but I wonder if adding `+S-waiting-on-review` unconditionally is a good idea, sometimes the PR is still being developed without needing a review and the author wants to run a perf. run.

Reply:
Hmm, not sure. Sometimes it's the right thing to do, sometimes not.